__init__.py

```diff
@@ -735,7 +737,7 @@ def handle_personal_backend_url(self, message):
         self.state = SetupState.PAIRING
         self.pairing.kickoff_pairing()
 
-    def handle_no_backend_selected(self, message):
+    def handle_no_backend_selected(self, _):
```

Comment:
why this change? please keep keyword argument, this helps in dig_for_message etc, for consistency we always want the message object to be explicit, not "_"

at any point that a message is received, we need to allow .forward/.reply

Reply:
Just resolving editor warnings, I'll change it back